### PR TITLE
docs - add GUC optimizer_enable_dml

### PR DESCRIPTION
**Pull request by mkiyama**
This will be backported to 5X_STABLE

See dev PR 5X_STABLE https://github.com/greenplum-db/gpdb/pull/8704

### CODE REVIEW

File: gpdb-doc/dita/ref_guide/config_params/guc-list.xml [hunk 1]
```diff
@@ -6874,13 +6876,50 @@
...
+    <body>
+      <p>When GPORCA is enabled (the default) and this parameter is <codeph>true</codeph> (the
+        default), GPORCA attempts to execute DML commands such as <codeph>INSERT</codeph>,
+          <codeph>UPDATE</codeph>, and <codeph>DELETE</codeph>. If GPORCA cannot execute the
+        command, Greenplum Database falls back to the Postgres Planner.</p>
+      <p>When set to <codeph>false</codeph>, Greenplum Database always falls back to the Postgres
+        Planner when performing DML commands.</p>
+      <p>The parameter can be set for a database system, an individual database, or a session or
+        query.</p>
+      <p>For information about GPORCA, see <xref
```
Comment (lisakowen) [inline gpdb-doc/dita/ref_guide/config_params/guc-list.xml]: do we need this xref in the guc description?  

not part of this PR:  there are several optimizer_xxx gucs.  some include xrefs to general gporca or query topics.  i don't see what value they are adding.

File: gpdb-doc/dita/ref_guide/config_params/guc-list.xml [hunk 2]
```diff
@@ -6874,13 +6876,50 @@
...
+    <title>optimizer_enable_dml</title>
+    <body>
+      <p>When GPORCA is enabled (the default) and this parameter is <codeph>true</codeph> (the
+        default), GPORCA attempts to execute DML commands such as <codeph>INSERT</codeph>,
+          <codeph>UPDATE</codeph>, and <codeph>DELETE</codeph>. If GPORCA cannot execute the
+        command, Greenplum Database falls back to the Postgres Planner.</p>
+      <p>When set to <codeph>false</codeph>, Greenplum Database always falls back to the Postgres
+        Planner when performing DML commands.</p>
+      <p>The parameter can be set for a database system, an individual database, or a session or
+        query.</p>
```
Comment (lisakowen) [inline gpdb-doc/dita/ref_guide/config_params/guc-list.xml]: "... for a session or a query" - doesn't setting it for the session set it for all queries in the session?  how/can you set this for a specific query?

**Comment by dyozie** [comment]
I think in addition to these changes, let's put [Accessing External Data with PXF](https://gpdb.docs.pivotal.io/6-0/admin_guide/external/pxf-overview.html) as the first option under [Working with External Data](https://gpdb.docs.pivotal.io/6-0/admin_guide/external/g-working-with-file-based-ext-tables.html) in the .ditamap.

File: gpdb-doc/dita/ref_guide/config_params/guc-list.xml (hunk 2 above)
Comment (mkiyama) [reply gpdb-doc/dita/ref_guide/config_params/guc-list.xml]: Yes, you can set a GUC for a specific query that overrides the session level guc.
I forget the specific syntax.

**Comment by mkiyama** [comment]
@dyozie Is this comment ^^^ in the correct PR?

File: gpdb-doc/dita/ref_guide/config_params/guc-list.xml (hunk 1 above)
Comment (mkiyama) [reply gpdb-doc/dita/ref_guide/config_params/guc-list.xml]: The xrefs were added when the having two query optimizers in the same database was new for GPDB. 

I am not sure how common this feature is in databases.

It still might be useful for new users to to know about this feature. I am not sure.

**Comment by dyozie** [comment]
> @dyozie Is this comment ^^^ in the correct PR?

Nope :(
